mlir/lib/Dialect/Math/Transforms/ExpandPatterns.cpp

```diff
@@ -33,6 +31,16 @@ static Value createFloatConst(Location loc, Type type, double value,
 
   return b.create<arith::ConstantOp>(loc, attr);
 }
+static Value createFloatConst(Location loc, Type type, APFloat value,
```

Comment:
I'd make the previous helper with `double` call into the new one with `APFloat` to reduce duplication

Reply:
Done.